app/brave_main_delegate_browsertest.cc

```diff
@@ -162,7 +162,7 @@ IN_PROC_BROWSER_TEST_F(BraveMainDelegateBrowserTest, DisabledFeatures) {
     &companion::features::internal::kSidePanelCompanion2,
     &companion::visual_search::features::kVisualSearchSuggestions,
     &content_settings::features::kUserBypassUI,
-    &dips::kFeature,
+    &features::kDIPS,
```

Comment:
Alphabetical order

Reply:
Thanks, fixed in 1302c260e7d46339d3c137a08e616a8cb8492e04.